BookingsAPI/Bookings.Infrastructure.Services/HearingDtoMapper.cs

```diff
@@ -0,0 +1,19 @@
+using System.Linq;
+using Bookings.Domain;
+using Bookings.Infrastructure.Services.Dtos;
+
+namespace Bookings.Infrastructure.Services
+{
+    public static class HearingDtoMapper
+    {
+        public static HearingDto MapToDto(Hearing hearing)
+        {
+            var @case = hearing.GetCases().First(); // Does this need to be a lead case?
+            return new HearingDto(hearing.Id, hearing.ScheduledDateTime, hearing.ScheduledDuration,
```

Comment:
Get rid of all CTORs and use property assignment in this mapper. IMHO, but up to you

Reply:
I was aiming to minimise the change since the dtos have ctors here. Usuaully  my dtos wouldn't have ctors but I've left it be for now